src/main/fc/config.h

```diff
@@ -75,6 +75,7 @@ typedef struct systemConfig_s {
     uint8_t activeRateProfile;
     uint8_t debug_mode;
     uint8_t task_statistics;
+    uint8_t rateProfile6PosSwitch;
 #if defined(STM32F4) && !defined(DISABLE_OVERCLOCK)
```

Comment:
The fact that `cpu_overclock` is included in the parameter group conditionally is ugly, and - like in the case at hand - will require the parameter group version to be incremented for any future addition of parameters. This is not desirable (in my opinion). Should we remove the conditionality from `cpu_overclock` in a separate pull request first, so that `rateProfile6PosSwitch` then can simply be appended?